### PR TITLE
Docs review (WIP)

### PR DESCRIPTION
**Pull request by atorin**
The purpose of this PR is to improve the docs while asking comments from the developers. 

The new document is available here: http://review-daf-docs.readthedocs.io/en/latest/

### CODE REVIEW

File: installation/local/devVM.rst [hunk 1]
```diff
@@ -2,26 +2,30 @@
...
 ******************
 
-This guide explaines how to use the virtual machine to create test enviroment.
+This guide explains how to use the Virtual Machine to create a test environment.
 
 You can download the OVA image at the following link:
 
 OVA Image: `download <https://developers.italia.it/static/DAF-Ubuntu16-Docker-test.ova>`_ (8.6 GB)
 
+.. warning::
```
Comment (atorin) [inline installation/local/devVM.rst]: I think it is important to mention this prerequisite. What do you think?

Comment (ruphy) [reply installation/local/devVM.rst]: Agreed. Hopefully we'll switch to Vagrant soon, maybe we should mention it? Or mention that we'd love PRs for this?

**Comment by ruphy** [comment]
make sure you also integrate PR #66

File: installation/local/devVM.rst (hunk 1 above)
Comment (atorin) [reply installation/local/devVM.rst]: OK, I will mention both things (Vagrant + we'd love PRs).

**Comment by atorin** [comment]
OK, before merging I will check that everything in #66 is indeed in this PR (some of that is already there). 

Also, we should consider whether to keep the last commit or not (b962a45). This requires that we agree on having a separate Italian manual, in a different document.

**Comment by atorin** [comment]
I am keeping the Italian user manual for the moment (I removed the last commit from the previous batch) and have implemented the requested changes. 

I think you can safely merge this PR now.

**Comment by atorin** [comment]
Given that we created a separate user manual in ITA, I have removed it from this document.

**Comment by ruphy** [comment]
merging given your input!

**Pull request merged**